test-lib-openshift.sh

```diff
@@ -195,7 +210,12 @@ function ct_os_wait_pod_ready() {
     echo -n "Waiting for ${pod_prefix} build pod to finish ..."
     while ! [ "$(ct_os_get_build_pod_status "${pod_prefix}")" == "Succeeded" ] ; do
       echo -n "."
-      [ "${SECONDS}" -gt "${timeout}0" ] && echo " FAIL" && return 1
+      [ "${SECONDS}" -gt "${timeout}0" ] && {
+        echo " FAIL";
+        ct_os_print_logs || :
+        ct_os_get_sti_build_logs "${pod_prefix}"
+        return 1
+      }
```

Comment:
I would rewrite it as an `if` now that it spans multiple lines to make it more readable. Same for the ones below

Reply:
```suggestion
      if [ "${SECONDS}" -gt "${timeout}0" ]; then
        echo " FAIL";
        ct_os_print_logs || :
        ct_os_get_sti_build_logs "${pod_prefix}"
        return 1
      fi
```